Provider/BaseVideoProvider.php

```diff
@@ -118,21 +118,33 @@ public function generatePrivateUrl(MediaInterface $media, $format)
      */
     public function buildEditForm(FormMapper $formMapper)
     {
+        // NEXT_MAJOR: Remove $textType variable and inline 'Symfony\Component\Form\Extension\Core\Type\TextType'
+        //             in $formMapper->add('binaryContent', ...)
+        $textType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
```

Comment:
Please do not use a variable for a single usage.

cc @greg0ire for the rule link. :wink: